gslib/utils/shim_util.py

```diff
@@ -607,6 +629,9 @@ def translate_to_gcloud_storage_if_requested(self):
       try:
         env_variables, gcloud_storage_command = self._get_full_gcloud_storage_execution_information(
             self.get_gcloud_storage_args())
+        if not self.gcloud_has_active_account():
```

Comment:
What if the boto file has credentials? Are those effectively ignored by the shim? If so, we should probably be clear about that, as I don't believe we specifically mention that in the shim documentation (except that the options are missing under the `[Credentials]` heading).

Reply:
Yes, anything in the boto file gets ignored as we are solely depending on the credentials used by gcloud (that's one of the conditions to use the shim, i.e gsutil and gcloud should be using the same creds.)  I think we have that mentioned for the internal docs, but for external docs, it's missing. I will file a bug.

Reply:
Actually, ended up creating a separate PR for the help text update: https://github.com/GoogleCloudPlatform/gsutil/pull/1756